HealthcareBase/Service/UsersService/UserFeedbackService/SurveyService/SurveyEntryService/RatedSectionService.cs

```diff
@@ -0,0 +1,174 @@
+using System.Collections.Generic;
+using System.Linq;
+using HealthcareBase.Model.Users.Survey.SurveyEntry;
+using HealthcareBase.Repository.Generics;
+using HealthcareBase.Repository.UsersRepository.SurveyRepository.SurveyEntryRepository.RatedSectionRepository;
+
+namespace HealthcareBase.Service.UsersService.UserFeedbackService.SurveyService.SurveyEntryService
+{
+    public class RatedSectionService : IRatedSectionService
+    {
+        private readonly RepositoryWrapper<RatedSectionRepository> ratedSectionRepository;
```

Comment:
Its better to use interface so you can satisfy Dependency injection

Reply:
RatedSectionRepository is, in fact, an interface but you can't tell by it's name.
I will fix it to fit interface naming convention in c#.